database-mongoose/seeder.js

```diff
@@ -0,0 +1,85 @@
+var faker = require('faker');
+let dbModel = require('./index.js');
+
+// var random = faker.name.findName();
+// console.log(random);
```

Comment:
Consider removing unneeded console.logs and comments

Reply:
Agree.